web/src/narrow_banner.js

```diff
@@ -333,6 +334,7 @@ function pick_empty_narrow_banner() {
                         ),
                     };
                 }
+                pm_list.show_empty_conversation_pm_list(user_ids);
```

Comment:
It feels like a bit of an abstraction boundary failure to have this called from here.

I would think that we should have `pm_list.handle_narrow_activated(current_filter);` be responsible for doing this work...

Reply:
Changed the code path, and made all the changes around `pm_list.handle_narrow_activated`, thanks for the suggestion.